app/src/main/res/layout/activity_bluetooth_connection.xml

```diff
@@ -18,17 +18,6 @@
             android:layout_height="match_parent"
             android:orientation="horizontal">
 
-            <ImageButton
```

Comment:
Oops, should have discussed this with you guys. I don't think that such a button is good style because I looked at all the Aegon, DeGiro, Rabobank, ABN AMRO and ING apps and all of them use the "ordinary" physical back button that's next to your big home button. If all big players are doing it that way, I think we should too (convention / more recognizable for user)